spec.md

```diff
@@ -1613,14 +1747,14 @@ Lookup of locals and globals may fail if not yet defined.
 
 ### Literals
 
-Starlark supports string literals of three different kinds:
+Starlark supports string literals of four different kinds:
 
 ```text
-Operand = int | float | string
+Operand = int | float | string | bytes
 ```
 
 Evaluation of a literal yields a value of the given type
-(int, float, or string) with the given value.
+(int, float, string, or bytes) with the given value.
```

Comment:
"given value" -> "denoted content"?

Reply:
"Evaluation of an int, float, string, or bytes literal yields the value of that literal."